vlada-skulska/README.md

```diff
@@ -0,0 +1,29 @@
+# Grammarly
+
+**Grammarly** is an online writing assistant that helps improve the quality of your written content. Whether you're writing an email, an essay, a blog post, or any other type of text, **Grammarly** provides valuable suggestions and corrections to enhance your writing.
+
+## Features
+
+- **_Grammar and Spelling:_** Grammarly detects grammar and spelling mistakes in your text and provides suggestions for improvement. It helps you avoid embarrassing errors and ensures your writing is clear and professional.
+
+- **_Clarity and Readability:_** Grammarly analyzes your writing style and offers suggestions to make your content more concise, readable, and engaging. It helps you eliminate wordiness, improve sentence structure, and maintain a consistent tone.
+
+- **_Vocabulary Enhancement:_** Grammarly suggests synonyms and alternative word choices to diversify your vocabulary and enhance the impact of your writing. It helps you find the right words to express your ideas effectively.
+
+- **_Plagiarism Detection:_** Grammarly's premium version includes a plagiarism checker that scans your text for similarities with millions of web pages. It helps you ensure the originality of your work and avoid unintentional plagiarism.
+
+## Getting Started
+
+To start using **Grammarly**, follow these steps:
+
+1. Sign up for an account on the [Grammarly website](https://www.grammarly.com/).
+
+2. Install the Grammarly browser extension or download the desktop application.
```

Comment:
For both of these there should be a separate section, diving a bit into details of each of the processes since functionality of extension and desktop app differs in terms of areas where text can be analyzed.